src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -77,10 +80,12 @@ case class BenchmarkResult(
     result: Option[Long] = None,
     breakDown: Seq[BreakdownResult] = Nil,
     queryExecution: Option[String] = None,
-    failure: Option[Failure] = None)
+    failure: Option[Failure] = None,
+    ml: Option[MLResult] = None)
```

Comment:
The test params should be stored separately from the result.  That way, when "ml" is not stored, the test params will still be available in the returned BenchmarkResult.